src/Geolocation.php

```diff
@@ -79,6 +79,9 @@ protected function doCall($parameters = array())
         // redefine response as json decoded
         $response = json_decode($response);
 
+        // API returns with an error
+        if (isset($response->error_message)) throw new GeolocationException($response->error_message);
```

Comment:
Can you change it to:

```
if (isset($response->error_message)) {
    throw new GeolocationException($response->error_message);
}
```

Reply:
Sure will do, although it breaks convention regarding the rest of the class :)